frontend/src/views/editUser/EditUser.jsx

```diff
@@ -0,0 +1,402 @@
+            <Box>
+              <Typography mb={1.5}>Status</Typography>
+              <RadioGroup
+                defaultValue="active"
+                style={{
+                  gap: 8,
+                  marginTop: 8,
+                }}
+                value={formData.active}
+                name="active"
+                onChange={handleChange}
+              >
+                <FormControlLabel
+                  value="active"
+                  control={<Radio />}
+                  label="Active, user can login to LCFS"
+                />
+                <FormControlLabel
+                  value="inactive"
+                  control={<Radio />}
+                  label="Inactive, user cannot login to LCFS"
+                />
+              </RadioGroup>
+            </Box>
+            <Box>
+              <Typography mb={1.5}>Roles</Typography>
+              {dummy.gov ? (
+                <>
+                  <FormControlLabel
+                    control={<Checkbox />}
+                    label="Administrator — can add/edit IDIR users and assign roles and add/edit organizations, BCeID users and assign roles"
+                    onChange={handleCheckbox}
+                    name="administrator"
+                    checked={formData.administrator}
+                  />
+                  <RadioGroup
+                    defaultValue="active"
+                    name="govRole"
+                    style={{
+                      gap: 8,
+                      marginTop: 8,
+                    }}
+                    onChange={handleChange}
+                    value={formData.govRole}
+                  >
+                    <FormControlLabel
+                      value="analyst"
+                      control={<Radio />}
+                      style={{
+                        '.MuiFormControlLabel-label': {
+                          fontSize: 16,
+                        },
+                      }}
+                      label="Analyst — can make recommendations on transfers, transactions and compliance reports, manage file submissions and add/edit fuel codes"
+                    />
+                    <FormControlLabel
+                      value="compliance_manager"
+                      control={<Radio />}
+                      label="Compliance Manager — can make recommendations on compliance reports"
+                    />
+                    <FormControlLabel
+                      value="director"
+                      control={<Radio />}
+                      label="Director — can assess compliance reports and approve transactions"
+                    />
+                  </RadioGroup>
+                </>
+              ) : (
+                <Stack spacing={1}>
+                  <FormControlLabel
+                    control={<Checkbox />}
+                    label="Manage Users — can add/edit BCeID users and assign roles"
```

Comment:
In the wireframe, the role titles are bold, but their descriptions are not. For example:

> **Manage Users** — can add/edit BCeID users and assign roles

Reply:
Mui limitation

Reply:
It should work:

```html
<FormControlLabel
  control={<Checkbox />}
  label={
    <span>
      <strong>Manage Users</strong> — can add/edit BCeID users and assign roles
    </span>
  }
  onChange={handleCheckbox}
  name="manageUsers"
  checked={formData.manageUsers}
/>
````

Reply:
good find 👍